### DISCUSSION TITLE
How to enhance the image's color

### DISCUSSION THREAD

**Discussion by Simon4638** [ImageMagick/ImageMagick#5353] [Help]
Dear Engineer:

I have same image from different Scanner. The issue that if we print images out with color printer, the output paper's color will look a little bit different. Some output paper may be a little bit red than others, some maybe darker than other. We would like using ImageMagick to get all image the same. Is there anyway to get this done by ImageMagic?

Any help will be appreciated.

**Comment by snibgo** [comment]
The proper way to do this is to calibrate your printer for each different paper. This will result in an ICC colour profile for each combination of printer and paper using, for example, ArgyleCMS. Then you can use ImageMagick to convert the image with the appropriate ICC profile before printing. 

A simpler method might work: choose one of the papers as the "most correct". Choose an image with a very wide range of colours. Print that on each paper, and scan them all. For each non-correct paper, find the transformation from that scanned image to the correct image. The transformation might be a simple gain-and-bias, or a clut that equalises the histograms, or something else.

Then before printing an image on a non-correct paper, use IM to apply the appropriate transformation.
